WordPress/src/main/AndroidManifest.xml

```diff
@@ -94,21 +94,6 @@
             android:theme="@style/Calypso.NoActionBar" />
 
         <!-- Account activities -->
-        <activity
-            android:name=".ui.accounts.SignInActivity"
-            android:configChanges="orientation|keyboardHidden|screenSize"
-            android:theme="@style/SignInTheme"
-            android:windowSoftInputMode="adjustResize">
-            <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
-                <category android:name="android.intent.category.DEFAULT" />
-                <category android:name="android.intent.category.BROWSABLE" />
-                <data
-                    android:host="signin"
```

Comment:
As @maxme has suggested over Slack, let's check with our #helpshift friends whether they need/use this shortcut to have people logged in. WDYT?

Reply:
I checked with #helpshift and it doesn't seem to be used so we can remove it.